airflow/models/baseoperator.py

```diff
@@ -635,6 +639,7 @@ class derived from this one results in the creation of a task object,
         runs across execution_dates.
     :param max_active_tis_per_dagrun: When set, a task will be able to limit the concurrent
         task instances per DAG run.
+    :param executor: which executor to target when running this task. NOT YET SUPPORTED
```

Comment:
Nittiest of nitpicks.  Some are capitalized, some aren't, feel free to ignore

```suggestion
    :param executor: Which executor to target when running this task. NOT YET SUPPORTED
```